docs/src/rules/no-loop-func.md

```diff
@@ -115,6 +115,34 @@ for (let i=10; i; i--) {
     a();
 }
 
+for (const i of foo) {
+    var a = function() { return i; }; // OK, all references are referring to block scoped variables in the loop.
+    a();
+}
+
+for (using i of foo) {
+    var a = function() { return i; }; // OK, all references are referring to block scoped variables in the loop.
+    a();
+}
+
+for (var i=10; i; i--) {
+	const foo = getsomething(i);
+    var a = function() { return foo; }; // OK, all references are referring to block scoped variables in the loop.
+    a();
+}
+
+for (var i=10; i; i--) {
+	using foo = getsomething(i);
+    var a = function() { return foo; }; // OK, all references are referring to block scoped variables in the loop.
+    a();
+}
+
+for (var i=10; i; i--) {
+	await using foo = getsomething(i);
+    var a = function() { return foo; }; // OK, all references are referring to block scoped variables in the loop.
+    a();
+}
```

Comment:
I think we will need to pass ECMA version 2026 in the code block to correctly open this in the playground.

Reply:
Doesn't it already work the same with the default `latest`?

Reply:
Yeah, that would work too. Anyway I also added 2026 as an option in https://github.com/eslint/eslint.org/pull/737